src/kansa/components/Exhibition.jsx

```diff
@@ -0,0 +1,548 @@
+    if(this.state.Works[i].id) {
+      console.log('work/'+this.state.Works[i].id)
+
+        raami.DELETE(`work/${this.state.Works[i].id}`).then(res=>{
+          console.log(res)
+    
+        })
+      } else {
+        alert('Cant delete nothing!')
+      }
+  }
+  
+  addWork() {
+    
+    var _work = this.state.Works.slice();
+
+      _work.push(
+          { id: null, 
+            artist_id: this.state.id, 
+            title: '',
+            width: '',
+            height: '', 
+            technique: '', 
+            orientation: '', 
+            filename: '', 
+            filedata: null, 
+            price: '',
+            year: '', 
+            gallery: ''
+          }
+      )
+
+    this.setState({Works: _work})
+    }
+
+  handleWork(i, field, e) {
+    var _work = this.state.Works.slice();
+    _work[i][field] = e.target.value; 
+    this.setState({Works:_work});    
+    
+  }
+
+  selectWork(i, field, e, key, val) {
+      var _work = this.state.Works.slice();
+      _work[i][field] = val; 
+      this.setState({Works:_work});
+    }
+
+
+  handleChange(field, e) {
+    var newState = {}; 
+    newState[field] = e.target.value; 
+    this.setState(newState);    
+    
+  }
+
+  handleCheck(field, e, val) {
+    var newState = {}; 
+    newState[field] = val; 
+    this.setState(newState);
+  }
+
+  handleSelect(field, e, key, val) {
+    var newState = {}; 
+    newState[field] = val; 
+    this.setState(newState);   
+  }
+
+  handleOpen = () => {
+    this.setState({open: true});
+    event.preventDefault()
+  };
+
+  handleClose = () => {
+    this.setState({open: false});
+  };
+
+  handleImage(i, e) {
+     // e.preventDefault();
+
+    var reader = new FileReader();
+    var file = e.target.files[0];
+    reader.onloadend = () => {
+
+          var _work = this.state.Works.slice();
+          _work[i]['filename'] = file.name;
+          _work[i]['filedata'] = reader.result;
+          this.setState({Works:_work});   
+      }
+    reader.readAsDataURL(file)
+    // console.log('image',this.state.Works[i])
+    }
+
+  handlePreview(e) {
+     // e.preventDefault();
+
+    var reader = new FileReader();
+    var file = e.target.files[0];
+    reader.onloadend = () => {
+
+      this.setState({
+        filename: file.name,
+        filedata: reader.result
+      });
+    }
+    reader.readAsDataURL(file)
+
+  }
+
+  render() {
+
+  /**** inline styles ****/
+
+  const grey = { 
+      color: '#888',
+      fontSize: '16px',
+      zIndex: '0'
+     }
+
+  const label = { 
+      color: '#888',
+      fontSize: '16px',
+
+     }
+
+  const paper = {
+      display: 'inline-block',
+      float: 'left',
+      padding: '20px',
+      marginTop: '20px',
+      marginLeft: '20px'
+  }
+
+  const zindex = {
+      zIndex: '0',
+      position: 'absolute'
+  }
+
+  const center = {
+    textAlign: 'center'
+  }
+
+  /**** calculate costs ****/
+
+
+  var works = []
+  
+  this.state.Works.forEach((work, i)=> {
```

Comment:
Replace with `const works = this.state.Works.map((work) => { ...` & use `work` rather than `this.state.Works[i]`.